ql/src/java/org/apache/hadoop/hive/ql/parse/repl/load/message/CreateDatabaseHandler.java

```diff
@@ -55,22 +55,23 @@ public List<Task<?>> handle(Context context)
     Database db = metaData.getDatabase();
     String destinationDBName =
         context.dbName == null ? db.getName() : context.dbName;
+    String destinationCatalogName = db.getCatalogName(); // TODO catalog. Need to double check the catalog here. Depend on HIVE-29278
```

Comment:
could we shorten `destinationCatalogName` -> `destCatalogName`. same for destDBName